src/cljs/rems/application.cljs

```diff
@@ -154,7 +154,9 @@
                              (rf/dispatch [::enter-application-page application-id]))
                          (rf/dispatch [::set-status {:status :failed
                                                      :description description
-                                                     :validation (:validation resp)}])))
+                                                     :validation prev-validation
+                                                     ;; this error is only shown if prev-validation was nil
```

Comment:
Hmm?

Reply:
if there are validation errors, the status-modal shows them and not the `:error`

Reply:
I think this part can be improved when we split the the different old actions to new files like the new actions are now.